Introduce performance tests and multi pipeline PoCs for metrics (#778)
kyma-project/community: concepts/observability-strategy/configurable-monitoring/pocs/tools/metric-stats/README.md

# metric-gen

The tool is used to analyze OTLP JSON metrics files, created by the [file exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/fileexporter).
It prints out the following statistics:
* types of the metrics that the files contain, along with the total number per metric type

> * max data point count 

Reviewer: ```suggestion
* Max data point count 
```

* max and average data point attribute count

## Usage

> One should extract the files created by the `file exporter`. There are multiple ways to do it. For example, add a sidecar with a shell to the OpenTelemetry Collector pod and share a volume with it, then execute `kubectl copy` to copy the files on your local machine.

Reviewer: ```suggestion
You can extract the files created by the `file exporter` in several ways. For example, add a sidecar with a shell to the OpenTelemetry Collector pod and share a volume with it, then execute `kubectl copy` to copy the files on your local machine.
```
